### PR TITLE
rbd: loosen image spec validation requirements

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+v10.2.0
```
Comment (jdurgin) [inline PendingReleaseNotes]: at this point you can modify doc/release-notes.rst directly for what's currently the jewel rc section

File: src/tools/rbd/action/ImportDiff.cc [hunk 2]
```diff
@@ -203,7 +203,7 @@ int execute(const po::variables_map &vm) {
   std::string snap_name;
   r = utils::get_pool_image_snapshot_names(
     vm, at::ARGUMENT_MODIFIER_NONE, &arg_index, &pool_name, &image_name,
-    &snap_name, utils::SNAPSHOT_PRESENCE_NONE);
```
Comment (jdurgin) [inline src/tools/rbd/action/ImportDiff.cc]: this still assumes the image already exists, so should be SPEC_VALIDATION_NONE

Comment (dillaman) [inline src/tools/rbd/action/ImportDiff.cc]: good catch -- I was in robot mode.

**Comment by dillaman** [comment]
@jdurgin updates pushed

**Comment by jdurgin** [comment]
lgtm, running through cli tests

**Pull request merged**